pkg/store/bucket_e2e_test.go

```diff
@@ -169,6 +169,7 @@ func prepareStoreWithTestBlocks(t testing.TB, dir string, bkt objstore.Bucket, m
 		true,
 		true,
 		DefaultPostingOffsetInMemorySampling,
+		false,
```

Comment:
I would just enable. This should work anyway, right? Thast's the point of compatibility (: